cogs/gatekeep/gatekeep.py

```diff
@@ -132,7 +136,36 @@ async def setThreshold(self, ctx: Context, threshold: int):
         else:
             await ctx.send("The value for the threshold should be greater than 0!")
 
-    @_gatekeep.command(name="days")
+    @settings.command(name="attachmentweight", aliases=["aw", "attachment", "attachments"])
+    @commands.guild_only()
+    @checks.mod_or_permissions(administrator=True)
+    async def setAttachmentWeight(self, ctx: Context, weight: int):
+        """Set the weight for how attachments in a message are scored.
+        Each attachment in a message will add this weight to the score when evaluating.
+
+        Parameters:
+        -----------
+        weight: int
+            Weight for which every attachment in a message will be scored.
+        """
+
+        if weight >= 0:
+            await self.config.guild(ctx.guild).get_attr(KEY_ATTACHMENT_WEIGHT).set(weight)
+            self.logger.info(
+                "%s#%s (%s) set the attachment weight to %s",
+                ctx.author.name,
+                ctx.author.discriminator,
+                ctx.author.id,
+                str(weight),
```

Comment:
I don't think `%s` is needed, but also we can just do `%d`, no?
```suggestion
                "%s#%s (%s) set the attachment weight to %d",
                ctx.author.name,
                ctx.author.discriminator,
                ctx.author.id,
                weight,
```

Reply:
Applied this change to attachment weight logs, as well as other logs that used %s and str(). All logs with numerical values should now use %d. The logs have also been updated to use ending periods, as there was some inconsistencies with how they were/weren't used.